libraries/localization-utilities/src/LocFileParser.ts

```diff
@@ -15,6 +20,7 @@ export interface IParseLocFileOptions {
   terminal: ITerminal;
   filePath: string;
   content: string;
+  parser?: ParserKind;
```

Comment:
Why would someone want to use a parser that's different from what's implied by the file extension?

Reply:
Now that I have exposed the parsers more directly, this feature here is less necessary. I had done this first before making each parser individually callable.
Easy example: the file is actually a different format altogether, that then gets converted by a separate loader into standard `.resjson` format.
I can revert this option, though, since if you know the format you should just call the individual parser directly and save the overhead of the dynamic dispatch.